test/functional/Java8andUp/src/org/openj9/test/gpu/SortTest.java

```diff
@@ -333,6 +333,10 @@ private static void unexpected(Exception e) {
 
 	private final Set<ElementType> types;
 
+	public SortTest( ) {
+		this(new String[0]);
+	}
+	
```

Comment:
Please remove the extraneous whitespace (between the parentheses and at the ends of lines).

Reply:
Done.